src/modules/teams.js

```diff
@@ -0,0 +1,220 @@
+  const url = `${API_BASE_URL}/teams/`;
+  return (dispatch, state) => {
+    return fetch(url, {
+      headers: {
+        Authorization: `Bearer ${state().user.token}`,
+        'Content-Type': 'application/json'
+      },
+      method: 'POST',
+      body: getBody(team)
+    })
+      .then((response) => {
+        if (response.ok) return response.json();
+        throw Error(response.statusText);
+      })
+      .then((response) => {
+        const team = response.data;
+        dispatch({
+          type: SET_EDITING,
+          payload: false
+        });
+        dispatch({
+          type: SET_LOADING,
+          payload: true
+        });
+        dispatch({
+          type: SAVE_TEAM,
+          payload: team
+        });
+        dispatch({
+          type: SET_LOADING,
+          payload: false
+        });
+        dispatch(getTeam(state().user.data.id));
+        if (state().teams.sendNotifications) {
+          dispatch({
+            type: SEND_NOTIFICATIONS,
+            payload: false
+          });
+          toastr.success('Request sent');
+        }
+        return team;
+      })
+      .catch((error) => {
+        dispatch({
+          type: SET_LOADING,
+          payload: false
+        });
+        console.warn('error', error)
+      });
+  };
+}
+export function updateTeam(team, id) {
+  const url = `${API_BASE_URL}/teams/${id}`;
+  return (dispatch, state) => {
+    return fetch(url,
+      {
+        headers: {
+          Authorization: `Bearer ${state().user.token}`,
+          'Content-Type': 'application/json'
+        },
+        method: 'PATCH',
+        body: getBody(team)
+      }
+    )
+      .then((response) => {
+        if (response.ok) return response.json();
+        throw Error(response.statusText);
+      })
+      .then((response) => {
+        const team = response.data;
+        dispatch({
+          type: SET_LOADING,
+          payload: true
+        });
+        dispatch({
+          type: SAVE_TEAM,
+          payload: team
+        });
+        dispatch({
+          type: SET_LOADING,
+          payload: false
+        });
+        dispatch(getTeam(state().user.data.id));
+        dispatch({
+          type: SET_EDITING,
+          payload: false
+        });
+        if (state().teams.sendNotifications) {
+          dispatch({
+            type: SEND_NOTIFICATIONS,
+            payload: false
+          });
+          toastr.success('Request sent');
+        }
+        return team;
+      })
+      .catch((error) => {
+        dispatch({
+          type: SET_LOADING,
+          payload: false
+        });
+        console.warn('error', error)
+      });
+  };
+}
+
+export function setEditing(value) {
+  return {
+    type: SET_EDITING,
+    payload: value
+  };
+}
+
+export function setLoading(value) {
+  return {
+    type: SET_LOADING,
+    payload: value
+  };
+}
+
+export function sendNotifications() {
+  return { 
+    type: SEND_NOTIFICATIONS, 
+    payload: true 
```

Comment:
you payload is always true, so the above functions do nothing as far as I can see.